Cli.py

```diff
@@ -180,6 +188,8 @@ def practice(self):
                 self.cursor_1.execute(shot_type_query, (self.practice_shot_type, ))
 
                 shot_type_record=self.cursor_1.fetchall()
+
+                #not sure where to put a lambda. But, I can probably put this as a list comp again
```

Comment:
List comp: see above
Lambda: Not really. Usually lambdas are used as a "just-in-time function" and less as a general function you can call. I use them a lot for the `map` built-in function, and they are useful for `sorted` too as a `key` function.

check this out for some clarity on why lambdas aren't appropriate in many cases: https://stackoverflow.com/questions/22985086/get-lambda-functions-name-as-string

Reply:
in other words, don't force the use of lambdas. You can still think about if it would be useful, but usually a standard function is best and more clear to others